e2e/teardown.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+docker kill iqengine
+docker kill iqenginedb
+docker network rm iqenginenet
+docker container prune --force
```

Comment:
This command might endup removing a lor more than iqengine itself. would be easier to do something like docker compose up and down ?

Reply:
Switched to docker compose. I'm curious - how exactly does this remove more than intended?

Reply:
Container prune will remove not only the containers of IQEngine in  case you had other containers running

Reply:
Since you are now using docker compose up wouldn't make sense to do a docker compose down in the teardown?

Reply:
yes